cmd/ssl-expiry-check/main.go

```diff
@@ -50,10 +54,34 @@ func init() {
 		log.Error("ERROR: The INSECURE environment variable has not been set.")
 		return
 	}
+	// set debug mode for nodeCheck pkg
+	nodeCheck.EnableDebugOutput()
 }
 
 func main() {
-	err := runExpiry()
+	// create context
+	checkTimeLimit := time.Minute * 1
```

Comment:
This works because the check runs quickly, but will bug out if the check takes longer than the `3m` `checkTimeout` that Kuberhealthy expects. Try using https://github.com/Comcast/kuberhealthy/blob/8889e96e6107513430ee533149cf6294b04b0119/pkg/checks/external/checkclient/main.go#L115 if you can

Reply:
GetDeadline functions for timeout checking added to both checks.